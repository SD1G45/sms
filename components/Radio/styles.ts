import styled from "styled-components";

const Radio = styled.input.attrs({
  type: "radio",
})`
  width: 20px;
  height: 20px;
  cursor: pointer;
`;

export default Radio;
