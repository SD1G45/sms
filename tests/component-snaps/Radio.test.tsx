import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Radio from "../../components/Radio";

describe("<Radio/>", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Radio/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});