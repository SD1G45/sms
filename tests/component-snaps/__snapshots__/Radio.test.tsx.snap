// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Radio/> renders correctly 1`] = `
.c0 {
  width: 20px;
  height: 20px;
  cursor: pointer;
}

<input
  className="c0"
  type="radio"
/>
`;
